src/components/modals/contacts-delete.test.tsx

```diff
@@ -0,0 +1,142 @@
+/*
+ * SPDX-FileCopyrightText: 2024 Zextras <https://www.zextras.com>
+ *
+ * SPDX-License-Identifier: AGPL-3.0-only
+ */
+import React from 'react';
+
+import { faker } from '@faker-js/faker';
+import { act } from '@testing-library/react';
+import { ErrorSoapBodyResponse } from '@zextras/carbonio-shell-ui';
+
+import { ContactsDeleteModal } from './contacts-delete';
+import { createSoapAPIInterceptor } from '../../carbonio-ui-commons/test/mocks/network/msw/create-api-interceptor';
+import { screen, setupTest } from '../../carbonio-ui-commons/test/test-setup';
+import { TESTID_SELECTORS } from '../../constants/tests';
+import { Contact } from '../../legacy/types/contact';
+import { buildContact } from '../../tests/model-builder';
+
+describe('ContactsDeleteModal', () => {
+	it('should render a modal with a specific title', () => {
+		const arrayContacts: Array<Contact> = [buildContact()];
+		setupTest(<ContactsDeleteModal contacts={arrayContacts} onClose={jest.fn()} />);
+		expect(screen.getByText('Are you sure to permanently delete this contact?')).toBeVisible();
+	});
+
+	it('should render a modal with a specific title', () => {
```

Comment:
This title should be different from the previous one as jest will recognize only one of those tests